internal/operator/operator_group.go

```diff
@@ -14,7 +14,7 @@ type OperatorGroupData struct {
 }
 
 func (o *operatorClient) CreateOperatorGroup(ctx context.Context, data OperatorGroupData, namespace string) (*operatorv1.OperatorGroup, error) {
-	logger.Debugf("Creating OperatorGroup %s in namespace %s", data.Name, namespace)
+	logger.Debugw("Creating OperatorGroup", data.Name, "Namespace", namespace)
```

Comment:
Debugw need to fix format; (msg string, keysAndValues ...interface{})
logger.Debugw("Creating OperatorGroup", "operatorgroup", data.Name, "namespace", namespace)?